packages/backend/src/peripherals/database/SyncStatusRepository.ts

```diff
@@ -0,0 +1,27 @@
+import type { KeyValueStore } from './KeyValueStore'
+
+/** block of the first verifier deploy */
+const EARLIEST_BLOCK = 11813207
+
+export type Store = Pick<KeyValueStore<'lastBlockNumberSynced'>, 'get' | 'set'>
+
+export class SyncStatusRepository {
+  private readonly earliestBlock: number
+
+  constructor(
+    private readonly store: Store,
+    options: { earliestBlock?: number } = {}
```

Comment:
why not?
```ts
    private readonly earliestBlock = EARLIEST_BLOCK,
```

Reply:
While refactoring SyncScheduler tests I noticed that having two consecutive numbers in constructor args is a bit error prone.


We had earliest block and batchSize, both with default values, so `new SyncScheduler(...objects, 1)` meant `earliestBlock=1, batchSize=500`. After the change it meant `batchSize=1`.

Reply:
Makes sense. Let's keep it this way then